ethereum/eth/build.gradle

```diff
@@ -55,4 +55,7 @@ dependencies {
   testImplementation 'org.mockito:mockito-core'
 
   jmhImplementation project(path: ':ethereum:core', configuration: 'testSupportArtifacts')
+
+  testImplementation project(path: ':config', configuration: 'testSupportArtifacts')
```

Comment:
This like should be alphabetized with the rest of the testImplementation dependenceies.  Between line 45 and 46

Reply:
Nice catch -- thank you.